.github/workflows/check-abi.yml

```diff
@@ -133,7 +131,7 @@ jobs:
       run: |
         ./abi-dumper.pl \
           ./head/build/lib/libze_loader.so \
-          -lver $(cat ./head/build/VERSION) \
+          -lver $(cat ./head/build/VERSION)-1 \
```

Comment:
What effect does adding this have?

Reply:
Without this the ABI tool had the same version for both versions it was checking which made it think it was checking the same version thus no differences. You need to set some version difference between the two otherwise it does not do a proper ABI check.